distribution/src/root/config/hazelcast-hardened.yaml

```diff
@@ -0,0 +1,233 @@
+# This is a sample Hazelcast configuration focused on hardened security.
```

Comment:
I would suggest to name this file `hazelcast-hardened-security.yaml`. Someone just browsing through the contents of the archive will recognize straight way it's focused on security.

Also I would add a note here at the beginning on how to use it (either rename to hazelcast.yaml or provide path to the config file in JAVA_OPTS or something).

Reply:
Will do :+1: